docs/platform/howto/restrict-access.md

```diff
@@ -5,40 +5,49 @@ sidebar_label: Restrict access
 
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-By default, Aiven services are publicly accessible, but you can restrict access to your service to a single IP, an address block, or any combination of both.
+Restrict access to your Aiven-managed service to a single IP, an address block, or any combination of both.
 
-:::important
-IP filters apply to inbound traffic coming from specified IP addresses / network ranges to
-Aiven service ports.
+By default, Aiven services are publicly accessible. To restrict access to Aiven services,
```

Comment:
Just realised that it's a bit too much to say that the services are publicly accessible by default. There is user/client authentication in place, so the service itself is not publicly accessible. But by default, connections to the service can be made from any address, after which the authentication takes place. But maybe this is obvious to the reader.